java/ql/lib/ext/experimental/org.springframework.core.io.model.yml

```diff
@@ -1,16 +0,0 @@
-extensions:
-  - addsTo:
-      pack: codeql/java-all
-      extensible: experimentalSinkModel
-    data:
-      - ["org.springframework.core.io", "ClassPathResource", True, "getFilename", "", "", "Argument[this]", "get-resource", "manual", "unsafe-url-forward"]
-      - ["org.springframework.core.io", "ClassPathResource", True, "getPath", "", "", "Argument[this]", "get-resource", "manual", "unsafe-url-forward"]
-      - ["org.springframework.core.io", "ClassPathResource", True, "getURL", "", "", "Argument[this]", "get-resource", "manual", "unsafe-url-forward"]
-      - ["org.springframework.core.io", "ClassPathResource", True, "resolveURL", "", "", "Argument[this]", "get-resource", "manual", "unsafe-url-forward"]
-  - addsTo:
-      pack: codeql/java-all
-      extensible: experimentalSummaryModel
-    data:
-      - ["org.springframework.core.io", "ClassPathResource", False, "ClassPathResource", "", "", "Argument[0]", "Argument[this]", "taint", "manual", "unsafe-url-forward"]
-      - ["org.springframework.core.io", "Resource", True, "createRelative", "", "", "Argument[0]", "ReturnValue", "taint", "manual", "unsafe-url-forward"]
-      - ["org.springframework.core.io", "ResourceLoader", True, "getResource", "", "", "Argument[0]", "ReturnValue", "taint", "manual", "unsafe-url-forward"]
```

Comment:
Same with these. Maybe they don't make sense anymore?

Reply:
Same as above for `ClassPathResource`.
`Resource.createRelative` was added as a `path-injection` sink in https://github.com/github/codeql/pull/14913.
A model for `ResourceLoader.getResource` [already exists](https://github.com/github/codeql/blob/8711232a388212846f71969b656f01f5269aecec/java/ql/lib/ext/org.springframework.core.io.model.yml#L7).

Reply:
It's fine as it is 👍